apps/web/src/services/my-page/my-account/user-account/modules/UserAccountModuleContainer.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <p-pane-layout class="form-wrapper">
```

Comment:
suggestion: I recommend to use unique class name in the root element.

```suggestion
    <p-pane-layout class="user-account-module-container">
```